tools/ci_build/build.py

```diff
@@ -1592,16 +1591,6 @@ def build_python_wheel(
         args = [sys.executable, os.path.join(source_dir, 'setup.py'),
                 'bdist_wheel']
 
-        # We explicitly override the platform tag in the name of the generated build wheel
-        # so that we can install the wheel on Mac OS X versions 10.12+.
```

Comment:
The warning makes sense and shouldn't be ignored. The core problem is we shouldn't link python.lib in non-Windows builds. It is unnecessary except when pyop feature is enabled. 

We don't enable pyop in the official builds. So it should be fine.